android-interop-testing/app/src/androidTest/java/io/grpc/android/integrationtest/InteropTesterTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2015, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.android.integrationtest;
+
+import static junit.framework.Assert.assertTrue;
+
+import android.support.test.runner.AndroidJUnit4;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(AndroidJUnit4.class)
+public class InteropTesterTest {
+  private final int TIMEOUT_SECONDS = 120;
+
+  @Test
+  public void interopTests() throws Exception {
+    final CountDownLatch latch = new CountDownLatch(1);
+    final AtomicBoolean passed = new AtomicBoolean(false);
+    new InteropTester(
+            "all",
```

Comment:
I agree that it looks a little strange, but this is the result of running google-java-format (double- and triple-checked this :)) so will leave it as-is.